releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,9 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.2 - July 01, 2020
+* Fixed the issue that caused a configuration change to be skipped as a part of refresh when a change is detected in a key with `refreshAll` set to `true` and all attempts to retrieve the updated settings from App Configuration fail. [#178](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/178)
```

Comment:
Feels like too much implementation detail. How about this?

"Fixed an issue that configuration may not be refreshed even though the key registered to refresh all configuration has changed."

Reply:
Sure, we can reduce the details. I think it might still be helpful to include that it only affects scenarios when `refreshAll:true` is set for a key. This will allow users who do not use it to know that they weren't impacted. How about this?

Fixed an issue that may cause configuration to not refresh on change in a key with `refreshAll: true`

Reply:
I was trying to be descriptive. "the key registered to refresh all configuration" is basically `refreshAll: true`. However, the latter is out of context. It's not clear what API we are talking about here. We cannot assume customers always know the context as us. You do have a linked issue that has sufficient implementation details, so I think a descriptive note works better in this case.

I see you prefer the style of "... issue that caused ...". I don't know which one is better :) Jimmy can help. Here is another version:

"Fixed an issue that may cause configuration refresh to be ignored even though the key registered to refresh all configuration has changed."